app/models/concerns/orchestration/compute.rb

```diff
@@ -37,7 +37,7 @@ def vm_name
 
   def queue_compute
     return unless compute? && errors.empty?
-    new_record? ? queue_compute_create : queue_compute_update
+    new_vm? ? queue_compute_create : queue_compute_update
```

Comment:
this will map a new host in foreman to an exists compute resource host, while I like this option I wonder if people would want that by default? (or at least not configurable)?